### PR TITLE
Add option to spawn IntroKit for those who randomspawn

### PR DESCRIPTION
**Pull request by Diet-Cola**

### CODE REVIEW

File: src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/NewfriendAssistConfig.java [hunk 1]
```diff
@@ -33,6 +33,10 @@
 	 * What is the newbie kit composed of.
 	 */
 	private ItemStack[] newbieKit;
+	/**
+	 * Should we give a kit to people who respawn without a bed/anchor?
+	 */
+	private boolean giveToRandomSpawners;
```
Comment (Maxopoly) [inline src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/NewfriendAssistConfig.java:39]: You're not actually reading this value from the config

**Review by Maxopoly — CHANGES_REQUESTED** [review]

File: src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/NewfriendAssistConfig.java (hunk 1 above)
Comment (Diet-Cola) [reply src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/NewfriendAssistConfig.java:39]: is this not what line 85 does?

Comment (Maxopoly) [reply src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/NewfriendAssistConfig.java:39]: I'm either blind or my diff bugged

File: src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/NewfriendAssistConfig.java [hunk 2]
```diff
@@ -78,6 +82,7 @@ protected void wireup(ConfigurationSection config) {
 		} else {
 			plugin().log(Level.INFO, " introkit disabled");
 		}
+		this.giveToRandomSpawners = config.getBoolean("giveIntroKitToRandomSpawners", true);
```
Comment (Diet-Cola) [inline src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/NewfriendAssistConfig.java:85]: @Maxopoly

**Comment by wingzero54** [comment]
I believe this is live on DevHell, is it good to go for us @Maxopoly ?  I'm not sure whether CivClassic will use it but I like staying up to date with the DevHell branch.

**Pull request merged**